pytim/gitim.py

```diff
@@ -199,24 +202,19 @@ def alpha_shape(self, alpha, group, layer):
             extraids = np.append(extraids, [-1] * n_cube)
         else:
             n_cube = 0
-            extrapoints = np.copy(points)
+            extrapoints = points
             extraids = np.arange(len(points), dtype=np.int)
 
-        self.triangulation.append(Delaunay(extrapoints))
-        try:
-            triangulation = self.triangulation[layer]
-        except IndexError:
-            raise IndexError("alpha_shape called using a wrong layer")
-        triangulation.radii = np.append(group.radii[extraids[extraids >= 0]],
-                                        np.zeros(8))
+        triangulation = Delaunay(extrapoints)
+        self.triangulation.append(triangulation)
 
         simplices = triangulation.simplices
+        radii = np.append(group.radii[extraids[extraids >= 0]],
+                          np.zeros(8))
 
-        try:
-            _points = self.triangulation[layer].points
-            radii = self.triangulation[layer].radii
-        except IndexError:
-            raise IndexError("alpha_shape called using a wrong layer")
+        # simply using extrapoints results in buffer dtype mismatch
```

Comment:
is this comment valid for the parent commit?

Reply:
No, the parent commit seems to be OK